crates/nu_plugin_polars/src/dataframe/command/core/open.rs

```diff
@@ -595,3 +626,20 @@ fn cloud_not_supported(file_type: PolarsFileType, span: Span) -> ShellError {
         inner: vec![],
     }
 }
+
+fn build_hive_options(call: &EvaluatedCall) -> Result<HiveOptions, ShellError> {
+    let enabled: Option<bool> = call.get_flag("hive-enabled")?;
+    let hive_start_idx: Option<usize> = call.get_flag("hive-start-idx")?;
+    let schema: Option<NuSchema> = call
+        .get_flag::<Value>("hive-schema")?
+        .map(|schema| NuSchema::try_from(&schema))
+        .transpose()?;
```

Comment:
Could this reuse the existing schema argument or is this something different?

Reply:
This is something different. Basically you can specify a schema on input and a schema you expect the results to look like from my understanding.